tests/Operation/AggregateFunctionalTest.php

```diff
@@ -12,6 +12,28 @@
 
 class AggregateFunctionalTest extends FunctionalTestCase
 {
+    public function testCurrentOpCommand()
+    {
+        if (version_compare($this->getServerVersion(), '3.6.0', '<')) {
+            $this->markTestSkipped('$currentOp is not supported');
+        }
+
+        (new CommandObserver)->observe(
+            function() {
+                $operation = new Aggregate(
+                    'admin',
+                    null,
+                    [['$currentOp' => (object) []]]
```

Comment:
I'm not specifying any options nor am I asserting the response as this is only testing that executing such a command with a `null` collection name results in `{ aggregate: 1 }` in the command document.